src/PermissionsContext.tsx

```diff
@@ -139,7 +139,7 @@ const PermissionsProvider: FunctionComponent = ({ children }) => {
   }
 
   const requestENPermission = async () => {
-    permissionStrategy.request()
+    permissionStrategy.request().catch(() => {})
```

Comment:
does the failure means something? Should we update any values?

Reply:
Plus one, swallowing errors is a code smell. do we know why this is a failed promise?

Reply:
My understanding is that this only happens when exposure notifications are requested on iOS, but the app is not authorized to share exposure information. So the attempt to enable ENs fails. I don't think we need to update anything if that is the case, but I may be missing something.

Reply:
So it might make sense to check if the app is iOS and authorized before making the request to enable app for en.

The problem with the generic catch(() => {}) is that if there ever is an unexpected error in the request, (like an api change or a typo), then we would have a very hard time debuging it or even finding out that it happened.

Reply:
Got it thanks @johnschoeman, this should fix it: https://github.com/Path-Check/gaen-mobile/pull/335/commits/b4f82c69008af7d2af97e367780c7a7cc79ca620